src/BookingForm.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import "./BookingForm.css";
+
+const BookingForm = props => {
+  const [newBooking, setNewBooking] = useState({
+    id: null,
+    firstName: "",
+    surname: "",
+    email: "",
+    roomId: null,
+    checkInDate: "",
+    checkOutDate: ""
+  });
+  const handleChange = e => {
+    const updatedBooking = { ...newBooking, [e.target.id]: e.target.value };
+    setNewBooking(updatedBooking);
+  };
+  const onSubmitHandle = e => {
+    e.preventDefault();
+    handleChange(e, props.addCustomer(newBooking));
```

Comment:
Some great logic for the form here - am not sure why you have to invoke `handleChange` with the `event` object when you are inside of the `onSubmitHandle` function